src/parsers/FR/FRMonthNameLittleEndianParser.js

```diff
@@ -14,7 +14,7 @@ var DAYS_OFFSET = util.WEEKDAY_OFFSET;
 
 var PATTERN = new RegExp('(\\W|^)' +
         '(?:(Dimanche|Lundi|Mardi|mercredi|Jeudi|Vendredi|Samedi|Dim|Lun|Mar|Mer|Jeu|Ven|Sam)\\s*,?\\s*)?' +
-        '([0-9]{1,2})' +
+        '([0-9]{1,2})(?:er)?' +

```

Comment:
Hi, yes I agree about the other days

I mainly followed what was done for other cultures ("2st" is legal value), but right, here in french it's only valid for the first day of month: "2er" would be wrong, but "21er" as well.

So it could probably changed to:

```
([0-9]{1,2}|1er)
```

Do you think the change is alright? 

And probably wouldn't require a refiner in this case